pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -68,6 +69,22 @@ type cniPortMapping struct {
 	HostIP        string `json:"hostIP"`
 }
 
+// cniBandwidthEntry maps to the standard CNI bandwidth Capability
+// see: https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md and
+// https://github.com/containernetworking/plugins/blob/master/plugins/meta/bandwidth/README.md
+type cniBandwidthEntry struct {
+	// IngressRate is the bandwidth rate in bps for traffic through container. 0 for no limit. If ingressRate is set, ingressBurst must also be set
+	IngressRate int `json:"ingressRate,omitempty"`
+	// IngressBurst is the bandwidth burst in b for traffic through container. 0 for no limit. If ingressBurst is set, ingressRate must also be set
+	// NOTE: it's not used for now.
```

Comment:
so it is required but not used?  Can't you just assume a default value of 0?

Reply:
> so it is required but not used?

Yes.

> Can't you just assume a default value of 0?

Okay.